src/components/Main/Main.js

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './Main.css';
+//Importing background to set background image// 
+import background from '../../background.png';
+import { animals } from '../../data';
+import Animal from '../Animal/Animal';
 
 export default function Main() {
-  return <main></main>;
+  return (<main className="App" style={{ backgroundImage: `url(${background})` }}>
+    {animals.map((animal) => (
+      <Animal key={animal.name} type={animal.type} says={animal.says} top={animal.top} left={animal.left} />
```

Comment:
Here we are not passing the property of name in the Animal component, which is why name is not rendering on the page.